docker/Dockerfile

```diff
@@ -0,0 +1,74 @@
+# Copyright 2015 Telefónica Investigación y Desarrollo, S.A.U
+#
+# This file is part of the IoT Agent for the Ultralight 2.0 protocol (IOTAUL) component
+#
+# IOTAUL is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the License,
+# or (at your option) any later version.
+#
+# IOTAUL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public
+# License along with IOTAUL.
+# If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by the GNU Affero General Public License
+# please contact with: [daniel.moranjimenez@telefonica.com]
+
+ARG NODE_VERSION=8.12.0-slim
+FROM node:${NODE_VERSION}
+ARG GITHUB_ACCOUNT=telefonicaid
+ARG GITHUB_REPOSITORY=iotagent-ul
+ARG DOWNLOAD_TYPE=latest
+
+ENV GITHUB_ACCOUNT=${GITHUB_ACCOUNT}
+ENV GITHUB_REPOSITORY=${GITHUB_REPOSITORY}
+ENV DOWNLOAD_TYPE=${DOWNLOAD_TYPE}
+
+MAINTAINER FIWARE IoTAgent Team. Telefónica I+D
+
+#
+# The following line retrieves the latest source code from GitHub.
```

Comment:
A couple of suggestions in the text to make it more clear:

The following line... -> The following RUN statement

Alternatively for local development, just copy this Dockerfile into file the root of the repository and copy over your local source using : -> Alternatively for local development, just copy this Dockerfile into file the root of the repository and replace the whole RUN statement by the following COPY statement in your local source using :

Reply:
Fixed - text added -  03e74ec